public/video-ui/src/components/VideoUsages/VideoUsages.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import {fetchUsages} from '../../services/capi';
+
+class VideoUsages extends React.Component {
+
+  state = {
+    usages: undefined
+  }
+
+  updateUsages(videoId) {
+
+    if (!videoId) {
+      return this.setState({
+        usages: undefined
```

Comment:
I did `[]` for the yt categories, shall we stick to one pattern? (happy to update the categories)